### DISCUSSION TITLE
Does Godog support using Scenario Outline with data from a CSV file?

### DISCUSSION THREAD

**Discussion by yogiis** [cucumber/godog#561] [Q&A]
Hello all,
 
I would like to know if Godog has a built-in feature or a way to use data from a CSV file in Scenario Outline. I want to be able to test scenarios with different combinations of data taken from a CSV file.

**Comment by roskee** [comment]
There is currently no implementation for that. You could create a function to read and parse your csv and add a step that calls the function - but hiding examples to an external file contradicts the original goal of having gherkin files- since they wont have a full meaning with the examples hiding in an external file.

**Reply by yogiis** [reply]
I see. Thank you for your answers.

I believe the best way to contribute is by directly adding support for Scenario Outline with CSV file to Godog
